From 43ad482936a4afa9935dd85b38c395d3d91ec36a Mon Sep 17 00:00:00 2001
From: Murray Cumming <murrayc@murrayc.com>
Date: Fri, 24 Jul 2015 07:52:11 +0200
Subject: Fix the previous commit

---
 tools/m4/filelist.am | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tools/m4/filelist.am b/tools/m4/filelist.am
index 38c37fb..0394790 100644
--- a/tools/m4/filelist.am
+++ b/tools/m4/filelist.am
@@ -21,4 +21,5 @@ files_codegen_m4 =		\
 	convert.m4		\
 	convert_gdk.m4		\
 	convert_gtk.m4		\
-	convert_gtkmm.m4
+	convert_gtkmm.m4 \
+  list.m4
-- 
cgit v0.11.2

